src/cascade/model/model.py

```diff
@@ -0,0 +1,115 @@
+from cascade.dismod.constants import WeightEnum
+from cascade.model.dismod_groups import DismodGroups
+from cascade.model.var import Var
+from cascade.model.smooth_grid import uninformative_grid_from_var
+
+
+class Model(DismodGroups):
+    """
+    A DismodGroups container of SmoothGrid.
+
+    Uses locations as given and translates them into nodes for Dismod-AT.
+    Uses ages and times as given and translates them into ``age_id``
+    and ``time_id`` for Dismod-AT.
+    """
+    def __init__(self, nonzero_rates, parent_location, child_location, covariates=None, weights=None):
+        """
+        >>> locations = location_hierarchy(execution_context)
+        >>> m = Model(["chi", "omega", "iota"], 6, locations)
+        """
+        super().__init__()
+        self.nonzero_rates = nonzero_rates
+        self.location_id = parent_location
```

Comment:
`parent_location`?

Reply:
Not sure what's confusing. There's always a parent location, right?

Reply:
Sorry, what I meant was: Should it be `self.parent_location` rather than `self.location_id` I think that would be more clear.